src/pipeline_func/clean.py

```diff
@@ -0,0 +1,266 @@
+###########################
+#     Initialize file     #
+###########################
+
+
+# For sample :
+
+# l.371 csv export
+# l.459 valid ids
+# engine sample path
+
+
+import pandas as pd
+import sys
+import glob
+import json
+import os
+from pathlib import Path
+import engine
+import numpy as np
+import scipy
+import datetime
+from scipy import stats
+
+ENGINE = engine.get_engine()
+ENGINE_rename = engine.get_engine()
+
+module_json = 'config/indicators.json'
+with open(module_json) as f:
+    for section in json.load(f):
+        for p in section['data']:
+            ENGINE[p['identifier']] = p['value']
+
+startTime = datetime.datetime.now()
+
+
+def make_note(statement):
+    print(statement, str(datetime.datetime.now() - startTime))
```

Comment:
This method is duplicated. Move it to separate helpers.py file and import when needed.